LWCOSC/force-app/main/default/lwc/invoiceItems/invoiceItems.html

```diff
@@ -0,0 +1,24 @@
+<template>
+    <lightning-card title="Invoice" icon-name="action:download">
+            <div>
+                <div class="invoice_header">
+                    <p>{invoice}</p>
+                </div>
+                <div class="slds-text-align_center">
+                    <p>{invoiceDate}</p>
+                </div>
+            </div>
+            <lightning-datatable
+                key-field="Id"
+                data={invoiceCartData}
+                columns={invoiceCartColumns}
+                hide-checkbox-column
+            >
+            </lightning-datatable><br/><br/><br/>
+            <lightning-button variant="brand" label="Place Order" 
```

Comment:
Must use custom labels instead of directly using texts/messages/labels

Reply:
Done